macos/FluentUI/Core/FluentUIResources.swift

```diff
@@ -5,17 +5,12 @@
 
 import AppKit
 
+
 /// A class for accessing the bundle and resource bundle associated with the FluentUI Framework
-public class FluentUIResources: NSObject {
-	/// The bundle of the FluentUI framework
-	static var bundle: Bundle { return Bundle(for: self) }
+class FluentUIResources: NSObject {
 	
-	/// The resource bundle contained within the FluentUI framework
-	@objc public static let resourceBundle: Bundle = {
-		guard let url = bundle.resourceURL?.appendingPathComponent("FluentUIResources-macos.bundle", isDirectory: true), let bundle = Bundle(url: url) else {
-			preconditionFailure("FluentUI resource bundle is not found")
-		}
-		
-		return bundle
+	/// The bundle of the FluentUI framework
+	static let resourceBundle: Bundle = {
```

Comment:
I need to preserve some fork of this logic for importing through cocoapods to actually still work.